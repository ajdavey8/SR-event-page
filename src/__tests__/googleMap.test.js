import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import GoogleMap from '../components/googleMap';


describe(GoogleMap, () => {
    // it('renders and matches snapshot', () => {
    //     const tree = renderer
    //       .create(<GoogleMap />)
    //       .toJSON();
    //     expect(tree).toMatchSnapshot();
    //   });
    it('should call component did mount method and call api', () => {
        const spy = jest.spyOn(GoogleMap.prototype, 'componentDidMount');
        const wrapper = mount(<GoogleMap />);
        wrapper.instance().componentDidMount();
        expect(spy).toHaveBeenCalled();
    })
})