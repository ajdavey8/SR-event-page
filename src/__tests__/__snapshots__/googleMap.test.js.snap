// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleMap renders and matches snapshot 1`] = `
<div
  style={
    Object {
      "height": "30vh",
      "width": "50%",
    }
  }
>
   
</div>
`;
